tests/api/v8/OAuth2Cest.php

```diff
@@ -51,15 +51,64 @@ public function TestScenarioInvalidClient(apiTester $I, \Helper\PhpBrowserDriver
         $I->seeResponseCodeIs(401);
     }
 
+    /**
+     * I want to make sure only the allowed grant types can be requested for any client
+     * @param apiTester $I
+     *
+     * HTTP Verb: POST
+     * URL: /api/oauth/access_token
+     */
+    public function TestScenarioGrantTypeNotAllowed(apiTester $I)
+    {
+        $client_id = 'API-ea74-c352-badd-c2be-5a8d9c9d4351';
+        $client_secret = 'secret';
+
+        $I->sendPOST(
+            $I->getInstanceURL().'/api/oauth/access_token',
+            array(
+                'grant_type' => 'password',
+                'client_id' => $client_id,
```

Comment:
consider using $I->getClientId() $I->getClientSecret() instead. Keeping the default value in one place is preferable, especially if we need to change it at a later time.